scenes/game/local_game.gd

```diff
@@ -3156,12 +3450,19 @@ func do_hit_response_effects(offense_player : Player, defense_player : Player, i
 	# If more of these are added, need to sequence them to ensure all handled correctly.
 	var events = []
 	active_strike.strike_state = next_state
-	if not offense_player.strike_stat_boosts.ignore_armor and defense_player.strike_stat_boosts.when_hit_force_for_armor:
-		change_game_state(Enums.GameState.GameState_PlayerDecision)
-		decision_info.player = defense_player.my_id
-		decision_info.type = Enums.DecisionType.DecisionType_ForceForArmor
-		decision_info.choice_card_id = active_strike.get_player_card(defense_player).id
-		events += [create_event(Enums.EventType.EventType_Strike_ForceForArmor, defense_player.my_id, incoming_damage)]
+	if not offense_player.strike_stat_boosts.ignore_armor:
+		# Currently assumes these will be armor-related; probably breaks if choices get involved
+		if active_strike.initiator == defense_player:
+			events += do_effects_for_timing("when_hit", defense_player, active_strike.initiator_card, next_state)
```

Comment:
I'm not a huge fan of passing in the card when it doesn't matter, but I see you're doing that as just a way to get the character effects....
Maybe just do what reshuffle_discard does?
var effects = get_character_effects_at_timing("on_reshuffle")
for effect in effects:
    events += parent.do_effect_if_condition_met(self, -1, effect, local_conditions)

I don't want to read this later and accidentally think it does more than it does.